examples/next/app-router/app/gated-content/page.js

```diff
@@ -0,0 +1,38 @@
+import { gql } from '@apollo/client';
+import { getAuthClient } from '@faustwp/experimental-app-router';
+
+export default async function Page() {
+  const client = await getAuthClient();
```

Comment:
When the `getAuthClient` is called and there is no refresh token, the client will return `null` so you can easily provide an unauthenticated state, like:

```js
 if (!client) {
     return <>You must be authenticated</>;
 }
```

Reply:
By default this would be `null` on this page. How do we really get this to authenticate?

Reply:
I just updated the testing instructions related to this.